### PR TITLE
Document history property in README.md

### PR DESCRIPTION
**Pull request by LukeeeeBennett**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -258,3 +258,24 @@ mock
...
+
+This is useful for testing.
+
+```js
+describe('Feature', () => {
+  it('requests an endpoint', () => {
+    var mock = new AxiosMockAdapter(axios);
+    mock.onGet('/endpoint').replyOnce(200);
+    
+    feature.request();
```
Comment (gitlab-winnie) [inline README.md]: @LukeeeeBennett This is probably an asynchronous call so you would either need a Promise / callback / await here. 😃

Comment (LukeeeeBennett) [reply README.md]: Doh, teaches me for getting too specific with examples. Fixing!

Comment (LukeeeeBennett) [reply README.md]: @gitlab-winnie How's it looking?

Comment (gitlab-winnie) [reply README.md]: Looks good to me! ✨

**Comment by ctimmerm** [comment]
Thanks!

**Pull request merged**